src/common/components/vanilla/button/button.css

```diff
@@ -99,6 +101,23 @@ $neutral-border-color: $dark-grey;
   background-color: $neutral-button-color-hover;
 }
 
+.link {
+  -webkit-appearance: none;
+  -moz-appearance:    none;
+  appearance:         none;
+  background-color: transparent;
+  border: 0;
+  color: $internal-button-text-color;
```

Comment:
We don't have variable called `$internal-button-text-color`, so this fails to compile.

Did you mean `$link-button-text-color` that you added in the top?